A Morse Code translator example.

jstitch opened the pull request:
Taken from '30 Arduino projects for the evil genius' book.

jstitch committed:
A Morse Code translator example.

Taken from '30 Arduino projects for the evil genius'.

theycallmeswift commented:
This is awesome.  I'm restructuring examples a bit, so I'll merge it with those changes.  Thanks for contributing!

jstitch committed:
Add RGB Led rgb and html methods.

rgb receives % of each R,G,B for the LED to emit
html receives a HTML color code and translates it to RGB

jstitch commented:
Added another commit, perhaps I should have waited for the first one to be pulled?
The other one is about RGB Leds...
